com.dz4k.FruitCredits.json

```diff
@@ -0,0 +1,3397 @@
+{
+  "id": "com.dz4k.FruitCredits",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.vala"
+  ],
+  "command": "fruit-credits",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--env=LD_LIBRARY_PATH=/app/lib:/app/lib64:/usr/lib/x86_64-linux-gnu/GL/default/lib:/usr/lib/x86_64-linux-gnu/openh264/extra:/usr/lib/x86_64-linux-gnu/openh264/extra:/usr/lib/sdk/vala/lib:/usr/lib/x86_64-linux-gnu/GL/default/lib"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/vala/bin",
+    "prepend-ld-library-path": "/usr/lib/sdk/vala/lib",
+    "env": {}
+  },
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "/share/vala",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "ghc-9.6.6-x86_64",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/bin/ghc",
+        "/bin/ghc-9.6.6",
+        "/bin/ghc-pkg",
+        "/bin/ghc-pkg-9.6.6",
+        "/bin/ghci",
+        "/bin/ghci-9.6.6",
+        "/bin/haddock",
+        "/bin/haddock-ghc-9.6.6",
+        "/bin/runghc",
+        "/bin/runghc-9.6.6",
+        "/bin/runhaskell",
+        "/bin/hp2ps",
+        "/bin/hpc",
+        "/bin/hsc2hs",
+        "/lib/libtinfo.so.5",
+        "/lib/debug",
+        "/lib/ghc-9.6.6",
+        "/share/man",
+        "/share/doc/ghc-9.6.6"
+      ],
+      "buildsystem": "simple",
```

Comment:
use autotools, put the `mkdir` and `ln -s` steps as `post-install`, remove the other two.